src/components/sections/FAQSection.astro

```diff
@@ -63,7 +63,7 @@ import ListItem from "../common/ListItem.astro";
                         Head to Terminal B Lower Level and take the <a href="https://www.flylax.com/flyaway-bus">FlyAway Bus</a> to <a href="https://metrolinktrains.com/rider-info/general-info/stations/l.a.-union-station/">Union Station</a>.
                     </ListItem>
                     <ListItem>
-                        Take the Metro A Line to <em>Memorial Park Station or Del Mar Station</em>, both stations are 0.4 mile away from the covnention center and about a ten minute walk.
+                        Take the Metro A Line to <em>Memorial Park Station or Del Mar Station</em>, both stations are 0.4 miles away from the covnention center and about a ten minute walk.
```

Comment:
I saw the `covnention` typo was fixed in https://github.com/nixcon/2024-na.nixcon.org/pull/37 so I left that out of here